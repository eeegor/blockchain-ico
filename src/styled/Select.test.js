import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Select } from '.';

describe('Select', () => {
	test('renders default select', () => {
		const tree = renderer
			.create(
				<Select>
					<option value="one">One</option>
				</Select>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
