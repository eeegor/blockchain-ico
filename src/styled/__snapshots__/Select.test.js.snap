// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select renders default select 1`] = `
.c0 {
  position: relative;
}

.c0::after {
  position: absolute;
  content: '';
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  top: 50%;
  right: 1rem;
  -webkit-transform: translateY(-30%);
  -ms-transform: translateY(-30%);
  transform: translateY(-30%);
  border: 6px solid transparent;
  border-color: #ebf0ff transparent transparent transparent;
}

.c0 select {
  width: 100%;
  -webkit-appearance: none;
  -moz-appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  border-radius: 4px;
  font-size: 13px;
  line-height: 24px;
  padding: 4px 36px 4px 12px;
  border: 1px solid #ebf0ff;
  background: #fff;
  position: relative;
  color: #2B60FF;
  font-family: GTCinetype-Bold;
}

.c0 select::-ms-expand {
  display: none;
}

<div
  className="c0"
>
  <option
    value="one"
  >
    One
  </option>
</div>
`;
